scripts/fullscreen.support.js

```diff
@@ -154,19 +155,20 @@
         if (!fullscreenEnabled()) {
             return;
         }
-        var images = document.getElementsByClassName('image');
+        var images = document.querySelectorAll('[data-tap-fullscreen]');
         for (var i = 0; i < images.length; i++) {
             var image = $(images[i]);
-            
+
             // Skip for images in slideshow component
-            if (image.closest('.slideshow').length !== 0) {
+            if (image.closest('[doc-slideshow]').length !== 0) {
                 continue;
             }
             // Skip individual disabled full-screen components
             if (image.hasClass('_disable-fullscreen')) {
                 continue;
             }
-            var url = image.find('div').css('background-image');
+            // Pick url of first found image
+            var url = image.find('[doc-image]').css('background-image');

```

Comment:
maybe also add addBack('[doc-image]') in case a user uses data-tap-fullscreen directly on a doc-image tag ?

Reply:
Done.